ReactWindows/ReactNative.Shared/Modules/Network/HttpContentHelpers.cs

```diff
@@ -1,27 +1,43 @@
 ﻿using System;
 using System.IO;
 using System.IO.Compression;
+#if WINDOWS_UWP
```

Comment:
Left comments here and we talked about them on video chat. All my feedback (aside from using PCLStorage) has been integrated.